### DISCUSSION TITLE
Problem with ru_core_news_sm-3.4.0 in spacy.load("ru_core_news_sm")

### DISCUSSION THREAD

**Discussion by ufo-government** [explosion/spaCy#11961] [Help: Installation] [answered]
Hi! I can not using ru_core_news_sm-3.4.0 (or any russian version).
Problem with ru_core_news_sm-3.4.0 in `spacy.load("ru_core_news_sm")`.
I have _AttributeError: module 'inspect' has no attribute 'getargspec'. Did you mean: 'getargs'?_  into stack python trace.
Ok! There is this subject https://github.com/pyinvoke/invoke/issues/833 .
Can you give me reccomendation about need Python version?

My env:
Python 3.11.0/spaCy v3.4.3/Windows  10.0.19044

**Comment by ljvmiranda921** [comment]
Hi @ufo-government , your spaCy version [should be able to support Python 3.11](https://pypi.org/project/spacy/#files). 
Can you show us the full stack trace instead? There might have been a problem in how spaCy was installed or some version in a dependency.

**Comment by adrianeboyd** [comment]
This is an issue we're aware of with python 3.11 and the third-party `pymorphy2` lemmatizer used in this model. See more details and workarounds in #11838.

**Comment by ufo-government** [comment, marked as answer]
Thanks for your answer guys!
@adrianeboyd, @ljvmiranda921 
best regards and good luck!

My choice:
RHCE 8 &
spaCy version    3.4.3
Location         /home/centos/py39env/lib64/python3.9/site-packages/spacy
Platform         Linux-4.18.0-305.7.1.el8_4.x86_64-x86_64-with-glibc2.28
Python version   3.9.13
Pipelines        ru_core_news_lg (3.4.0)
